pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -832,6 +837,12 @@ func (ssc *defaultStatefulSetControl) refreshPodState(set *appsv1beta1.StatefulS
 
 	var state appspub.LifecycleStateType
 	switch lifecycle.GetPodLifecycleState(pod) {
+	case appspub.LifecycleStatePreparingNormal:
```

Comment:
There are two things we need to consider further:

1. This state transition is invoked after successfully scaling. Assuming that the scale is stuck,  this transition will also be stuck. Is it reasonable?

2. If a pod-x has not entered Normal state yet after creating,  should we continue to create pod-x+1?